iina/PrefKeyBindingViewController.swift

```diff
@@ -101,10 +101,12 @@ class PrefKeyBindingViewController: PreferenceViewController, PreferenceWindowEm
 
     useMediaKeysButton.title = NSLocalizedString("preference.system_media_control", comment: "Use system media control")
 
-    // Load the config file saved in user default
-    loadConfigFile(Preference.string(for: .currentInputConfigName), true)
+    DispatchQueue.main.async {
+      // Load the config file saved in user default
+      self.loadConfigFile(Preference.string(for: .currentInputConfigName), true)
```

Comment:
I like the new changes. Why queue the initialization and not just dispatch the alerts?

My concern is that initialization that used to be completed in `viewDidLoad` is now delayed making initialization technically non-deterministic. I'm not sure this is a problem in practice, but to demonstrate the potential problem I changed the dispatching to have a delay:
```swift
    DispatchQueue.main.asyncAfter(deadline: .now() + 10) {
```

The panel allowed me to modify the `IINA Default` key bindings because the initialization had not finished:
![modified](https://github.com/user-attachments/assets/23325353-8c82-42fa-9214-0a981ba95db6)

I do not know this part of the IINA code. Lets see if @uiryuu has any concerns or thinks these changes are good to go.

Reply:
I think it's fine, since `DispatchQueue.main.async()` should run the block as soon as the main thread is available, before any `main.async()` posted later and any possible user interactions. This is different from `DispatchQueue.main.asyncAfter(deadline: .now() + 10)` where you can have any kind of stuff during the 10 seconds.

However, I prefer to only dispatch the `showAlert` call, since dispatching the whole initialization will add unnecessary complexity.  Even if the initialization works fine asynchronously right now, its behavior can change in the future and may cause issues.